operations/patch.go

```diff
@@ -178,8 +178,8 @@ func Patch() cli.Command {
 			}
 			params.Description = params.getDescription()
 
-			if (params.RepeatDefinition || params.RepeatFailed) && (len(params.Tasks) > 0 || len(params.Variants) > 0) {
-				return errors.Errorf("can't define tasks/variants when reusing previous patch's tasks and variants")
+			if (params.RepeatDefinition || params.RepeatFailed) && (len(params.Tasks) > 0 || len(params.Variants) > 0 || len(params.Alias) > 0) {
```

Comment:
Actually, I thought it over again - does this have to check RegexVariants and RegexTasks are both empty as well? They might be populated if they specify a local alias.

Reply:
good catch! fixed.